angular/src/app/directives/leaflet/leaflet.directive.ts

```diff
@@ -0,0 +1,41 @@
+import { Directive, ElementRef, Input, NgZone } from '@angular/core';
+import L from 'leaflet';
+import { MapCrs } from '../../services/map-crs';
+
+const mapOptions = {
```

Comment:
Het is denk ik wel netjes om de config in losse files te stoppen en te importeren, wat vind jij?